tests/e2e_tests/multistep/test_incentive.py

```diff
@@ -120,13 +120,9 @@ async def test_incentive(local_chain):
 
     # Create tasks to read stdout and stderr concurrently
     # ignore, don't await coroutine, just write logs to file
-    await asyncio.create_task(
-        write_output_log_to_file("miner_stdout", miner_process.stdout)
-    )
+    asyncio.create_task(write_output_log_to_file("miner_stdout", miner_process.stdout))
     # ignore, dont await coroutine, just write logs to file
-    await asyncio.create_task(
-        write_output_log_to_file("miner_stderr", miner_process.stderr)
-    )
+    asyncio.create_task(write_output_log_to_file("miner_stderr", miner_process.stderr))
```

Comment:
These need to be awaited.

Reply:
According to Daniel, he uses this for debugging, so he asked to ignore it.